src/components/Inputs/CustomLookup.js

```diff
@@ -265,12 +267,16 @@ const CustomLookup = ({
             placeholder={secondFieldLabel == '' ? displayField.toUpperCase() : secondFieldLabel.toUpperCase()}
             value={secondValue ? secondValue : ''}
             required={isRequired}
-            disabled={disabled}
+            onChange={e => {
+              if (onSecondValueChange && secondFieldName) {
+                onSecondValueChange(secondFieldName, e.target.value)
+              }
+            }}
             InputProps={{
               inputProps: {
-                tabIndex: -1 // Prevent focus on the input field
+                tabIndex: _readOnly ? -1 : 0 // Prevent focus on the input field
               },
-              readOnly: true
+              readOnly: !!secondFieldName ? false : true
```

Comment:
What happens if the value of _readOnly is equal to true?

Reply:
we want the field as editable